llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -5449,7 +5448,7 @@ static bool removeEmptyCleanup(CleanupReturnInst *RI, DomTreeUpdater *DTU) {
     }
 
     // Sink any remaining PHI nodes directly into UnwindDest.
-    Instruction *InsertPt = DestEHPad;
+    BasicBlock::iterator InsertPt = UnwindDest->getFirstNonPHIIt(); // XXX unsafe faff
```

Comment:
Comment leftover from development? And why is it unsafe?

Reply:
Left over indeed, removing -- it was uh, "unsafe" because I thought some of these call-sites wouldn't recompile cleanly without touching headers, and I was on a laptop which didn't have enough energy left to totally recompile LLVM X_X.